src/ros_filter.cpp

```diff
@@ -739,6 +740,9 @@ namespace RobotLocalization
     // Whether we're publshing the world_frame->base_link_frame transform
     nhLocal_.param("publish_tf", publishTransform_, true);
 
+    // Whether we're publshing the the inverse transfrom, ie. base_link_frame-world_frame transform
```

Comment:
Same typo as before. Actually it seems you copied the lines from above and then edited them, since the typo is also in them